lms/djangoapps/dashboard/decisions/0002-deprecate-sysadmin-dashboard-adr

```diff
@@ -0,0 +1,45 @@
+1. Deprecating the Sysadmin Dashboard
+---------------------
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Maintaining the sysadmin dashboard is challenging, and it is not widely used. The code is part of the lms
+application, even though most of it's use cases are relevant to course authoring.
+
+The sysadmin dashboard would be better suited as a pluggable django application, using appropriate APIs in the
+cms application
+
+Decision
+--------
+
+In order to deprecate the sysadmin dashboard and move it to a pluggable django application, the followings APIs
+would need to be added and/or moved into the cms application
+
+1. Create a cms user account
```

Comment:
Curious: Is there a reason CMS user creation needs to be done from a separate sysadmin dashboard?  Is this a workflow not supported elsewhere in the platform?

Reply:
I have the same question. =)

There's a chicken-or-the-egg UX challenge with the current interfaces. As an admin, I want to provision courses for faculty who have never used the platform before. I can create the course, but I can't give them CMS access unless they have an account. Account creation is only self-service. So I have to go through a mutli-step process of inviting the faculty, waiting for them to complete account creation, and only than can I gave them access to the course in studio. I'd like to do that in a single step.